#request
int32 path_length
---
#response
int8 ingore_this

